src/main/java/org/spongepowered/common/event/tracking/phase/packet/player/InteractionPacketState.java

```diff
@@ -84,6 +84,7 @@ public boolean isInteraction() {
 
     @Override
     public void populateContext(final EntityPlayerMP playerMP, final Packet<?> packet, final InteractionPacketContext context) {
+        ((TrackedInventoryBridge) playerMP.openContainer).bridge$setCaptureInventory(true);
```

Comment:
I don't feel comfortable setting the inventory to capture blindly like this, are we for certain that this doesn't negatively interact with other inventory related captures that might take place when interacting with other bits (like right clicking a water bucket to a cauldron)